### ISSUE TITLE
[Feature Request] Config setting to disable options in webUI

### ISSUE THREAD

**Issue by DeviantEng** [AHAAAAAAA/PokemonGo-Map#3027]
With the increase of added option in the web frontend, I would like to suggest the ability to disable them completely within the config (file or runserver.py option).  With a setup scanning multiple areas (or one large area), the ability to kick off a search from the web frontend is undesirable, IMO. 

## Possible Solution
Config option to disable/remove Options from web frontend (i.e., `Start at user's location`, `Search`, `Icons`, `Icon Size`, etc), similar to how `-fl` removes/disables the location field.

**Comment by timlinde** [comment]
Just edit the following files and you can easily create your own options on what you want to hide or not.

https://github.com/AHAAAAAAA/PokemonGo-Map/blob/develop/templates/map.html#L71
https://github.com/AHAAAAAAA/PokemonGo-Map/blob/develop/pogom/utils.py#L76
https://github.com/AHAAAAAAA/PokemonGo-Map/blob/develop/pogom/app.py#L64
https://github.com/AHAAAAAAA/PokemonGo-Map/blob/develop/pogom/app.py#L72

Just copy and paste the structure but rename the variables to whatever you want to call your options.
Hope this helps a bit.

**Comment by DeviantEng** [comment]
@timlinde Thanks for the links.  I considered doing something like this based on using `-fl` as an example, but I was trying to avoid doing custom changes because they'll revert whenever I do a `git pull`.

**Comment by timlinde** [comment]
@DeviantEng you need to fork the development branch. Then you do the following changes i suggested and commit them. Then you can easily pull new new changes without loosing your own changes made to the project.

Tell me if anything is unclear.

**Comment by DeviantEng** [comment]
@timlinde My git experience has generally been limited to `clone`, `pull`, `add`, `commit`, and `push`.  It makes sense what you're saying, but I've never done it from git cli so I'm trying to wrap my brain around how exactly to do it.  

I'm guessing I need to create a new repo, git pull from this project (develop branch), make my changes, commit and push to my repo.  if I ever want to pull from this repo (upstream), then I just run `git pull https://github.com/AHAAAAAAA/PokemonGo-Map.git develop` and resolve any conflicts?  Does that sound right?

**Comment by timlinde** [comment]
@DeviantEng No problem, gotta learn sometime right?

There is some useful information here as well:
http://stackoverflow.com/questions/3903817/pull-new-updates-from-original-github-repository-into-forked-github-repository

Just give me a shout if you need help.

**Comment by DeviantEng** [comment]
Thanks!  I never realized that it was so simple with github, but I've already forked over to my own develop branch and added a few commit of my customizations (branding, adsense, google analytics).  

From my understanding, all I now have to do is run `git fetch upstream`, `git checkout develop`, and `git merge upstream/develop` to pull down any new commits from the main repo.
Do I then need to run git push to add those new commits from `upstream`, to `origin` (my new repo)?

**Comment by timlinde** [comment]
I don't think you want to checkout develop. You only want to merge it to your "working branch" (origin)

You should not be required to push the changes.

**Comment by timlinde** [comment]
@DeviantEng You are right. I must be getting tired... Haha.
It might be good to push the changes yes :)
